js/ion.rangeSlider.js

```diff
@@ -1621,6 +1650,16 @@
         },
 
         /**
+         * Convert percent to its reverse
+         *
+         * @param value {Number} X in percent
+         * @returns {Number} X in reverse percent
+         */
+        convertToReversePercent: function (value) {
+            return this.options.reverse ? this.toFixed(100 - value) : value;
```

Comment:
No need to check this.options.reverse in here. You should do direct calculation.